modules/oxide/Gruntfile.js

```diff
@@ -53,8 +53,32 @@ module.exports = function (grunt) {
           livereload: true
         }
       }
-    }
-  });
+    },
+    // Process and add styles to the End repository
+    'string-replace': {
+      copyToEnd: {
+        files: [
+          {
+            expand: true,
+            cwd: './build/skins/ui/wagtail/',
+            src: ['skin.css', 'content.min.css'],
+            dest: '../../../end/src/assets/tinymce/'
+          },
+          {
+            expand: true,
+            cwd: './build/skins/ui/wagtail-dark/',
```

Comment:
For a clean commit history, the dark theme should not be part of this PR, it does not exist yet :wink: